packages/service-library/src/servicelib/redis.py

```diff
@@ -61,6 +68,9 @@ def __post_init__(self):
             decode_responses=True,
         )
 
+    async def _check_redis_health(self) -> None:
+        self._is_redis_healthy = await self.ping()
```

Comment:
MINOR: by default I assume that all protected members of `RedisClientSDK` refer to redis, therefore, I would drop `redis` from the name of all protected attributes and members